Makefile

```diff
@@ -1,8 +1,11 @@
 SHELL := /bin/bash
 NRI_GENERATOR_PATH="$(PWD)/nri-config-generator"
 
+GOOS ?=
+GOARCH ?=
+
 ifeq (, $(shell which newrelic-integration-e2e))
-NEWRELIC_E2E ?= go run github.com/newrelic/newrelic-integration-e2e-action/newrelic-integration-e2e/cmd@v1
+NEWRELIC_E2E ?= go run github.com/newrelic/newrelic-integration-e2e-action/newrelic-integration-e2e/cmd@latest
 else
 NEWRELIC_E2E ?= newrelic-integration-e2e
```

Comment:
does it make sense to keep the installed command? I fear this can create issues

Reply:
I think until we fully fix the go run command yes... It's what I am doing right now. The `go run`  for any reason is pointing to an old version of the action, not the latest one.